src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -203,7 +203,18 @@ describe('non-empty successful affected clusters table', () => {
     });
   });
 
+  it('sorting the last seen column', () => {
+    cy.get(TABLE)
+      .find('td[data-key=2]')
+      .children()
+      .eq(0)
+      .should('have.text', '2 years ago');
+    cy.get('.pf-c-table__sort').eq(1).click();
```

Comment:
What's the point of this action? The same for line 223.

Reply:
removed it, the next test is testing everything we need